Makefile

```diff
@@ -15,7 +15,7 @@ SYSTEM_DEPENDENCIES= \
 	wget
 OS=$(shell lsb_release -si)
 PYTHON_MAJOR_VERSION=3
-PYTHON_MINOR_VERSION=6
+PYTHON_MINOR_VERSION=$(shell python -c 'import sys;print(sys.version_info[1])')
```

Comment:
I see the idea, and indeed we could probably improve that Makefile somehow, but this is currently hardcoded somehow on purpose.
Because for instance on Ubuntu 18.04 LTS default system Python is 2.7 (shame!). The purpose of this Makefile is to prepare the virtualenv. Do you see what I mean?
However I understand the need of wanting it to automatically prepare a Python3.7 environment for you without having to edit the Makefile. In this case you can use the built-in variable overriding mechanism of Makefile, e.g. via:
```
make PYTHON_MINOR_VERSION=7
```
That will do just what you need.